notebooks/official/tabular_workflows/wide_and_deep_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1096 @@
+{
```

Comment:
<p>We -&gt; You</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/984/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>Done.</p>